### ISSUE TITLE
Add support for organizationNameKey for oauth token exchange

### ISSUE THREAD

**Issue by omgitstom** [stormpath/stormpath-sdk-php#161]
https://stormpath.atlassian.net/browse/AM-3287

**Comment by bretterer** [comment]
Ability to add orgNameKey to PasswordGrantRequest.  The access_token returned will include org claim if a valid organization was provided. 

```
$passwordGrant = new \Stormpath\Oauth\PasswordGrantRequest(self::$account->username, $password, self::$org->nameKey);

		$auth = new \Stormpath\Oauth\PasswordGrantAuthenticator(self::$application);
		$token = $auth->authenticate($passwordGrant);
```

**Comment by mdeggies** [comment]
@bretterer Hey Brian -- Do you have any timeline on this? We have a customer who has been asking us about this feature for 6 months now, and needs an ETA.

**Comment by bretterer** [comment]
This is merged into dev and ready to release in 1.18.0
